plugins/kotlin/j2k/shared/tests/testData/newJ2k/stream/stringLines.java

```diff
@@ -0,0 +1,8 @@
+// IGNORE_K2
```

Comment:
With K2, the generated method is `fun foo(s: String): Stream<String?>` and because of it the test is failing.

Reply:
That is to be expected, because K2 J2K type argument nullability inference is not implemented yet.